src/scicookie/{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -40,6 +40,18 @@ repos:
       types:
         - python
 {% endif %}
+{%- if cookiecutter.use_ruff == 'ruff' %}
+    - entry: ruff
+      files: ""
+      id: ruff
+      language: python
+      name: ruff
+      pass_filenames: true
+      stages:
```

Comment:
you don't need to define stages here, because it is already defined in line 6 via `default_stages`